packages/grid/helm/syft/values.yaml

```diff
@@ -159,15 +159,18 @@ node:
   rootEmail: info@openmined.org
   type: domain
   side: high
-  inMemoryWorkers: false
-  defaultWorkerPoolCount: 1
   defaultBucketName: null
   queuePort: 5556
   logLevel: info
   debuggerEnabled: false
   associationRequestAutoApproval: false
   useInternalRegistry: true
 
+  # Worker pool settings
+  inMemoryWorkers: false
+  defaultWorkerPoolCount: 1
```

Comment:
I think we should split this into

```yaml
defaultWorkerPool:
    tag: null
    count: 1
    podAnnotations: null
    podLabels: null
```

thoughts?

Reply:
Addressed in [9692b6e](https://github.com/OpenMined/PySyft/pull/8884/commits/9692b6e8cbad38bf272f5e66761b6a95b1337e94)